bazel/container_structure_test.bzl

```diff
@@ -29,7 +29,7 @@ readonly yq=$(rlocation {yq_path})
 
 # When the image points to a folder, we can read the index.json file inside
 if [[ -d "{image_path}" ]]; then
-  readonly DIGEST=$("$yq" eval '.manifests[0].digest | sub(":"; "-")' "{image_path}/index.json")
+  readonly DIGEST=$("$yq" eval -oj -r '.manifests[0].digest | sub(":"; "-")' "{image_path}/index.json")
```

Comment:
nit: maybe some comments about what the flags are for?

Reply:
sure. i'll do that.

Reply:
done